src/language-js/print/typescript.js

```diff
@@ -52,9 +52,24 @@ const {
 
 function printTypescript(path, options, print) {
   const node = path.getValue();
+
+  // TypeScript nodes always starts with `TS`
+  if (!node.type.startsWith("TS")) {
+    return;
+  }
+
+  if (/^TS[A-Z][a-z]+Keyword$/.test(node.type)) {
```

Comment:
```suggestion
  if (node.type.endsWith("Keyword")) {
```

Reply:
I was hoping to only matches ONE word keyword, not things like `TSBigIntKeyword`. We don't know future two words keyword should be camel case or not.

Reply:
There is no camel case keywords in JS and TS. Things like `instanceof` are written all lowercase.

Reply:
Edited in the web editor https://github.com/prettier/prettier/pull/10869/commits/f801f5824c43466c058388b6706f94332067d6fb